packages/auto-approve/test/auto-approve.test.ts

```diff
@@ -140,7 +148,12 @@ describe('auto-approve', () => {
         ));
 
         const scopes = [
-          getConfigFile('testOwner', 'testRepo', 'fake-config', 200),
+          getConfigFile(
```

Comment:
In the future, I would call these `stubGetConfigFile` or `mockGetConfigFile` to clarify that it's stubbing, not actually getting a config file.